yarn.lock

```diff
@@ -0,0 +1,12118 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do not check in lock files.

Reply:
Should I add it in `.gitignore`?

Reply:
It's already added. You may have to get latest from main.